testing/telegram-tdlib/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=telegram-tdlib
+pkgver=1.4.0
+pkgrel=0
+pkgdesc="Cross-platform library for building Telegram clients"
+url="https://core.telegram.org/tdlib"
+arch="all"
+license="BSL-1.0"
+options="!check" # No test suite from upstream
+depends=""
```

Comment:
empty variables can be removed for brevity

Reply:
Fixed. Thank you! Really missed it, it supposed I already know that :-)